code/modules/urist/GOAI/_datastructures/lists.dm

```diff
@@ -0,0 +1,139 @@
+/*
+ * Holds procs to help with list operations
+ */
+
+
+
+/proc/upsert(var/list/oldassoc, var/list/newassoc)
```

Comment:
This and some others are unused, will nuke. Holdover from veeeeery long time ago.